<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BluetoothDialog</class>
 <widget class="QDialog" name="BluetoothDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Bluetooth</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QPushButton" name="bltPowerPushButton">
     <property name="text">
      <string>Turn On</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="visPushButton">
     <property name="text">
      <string>Set visible</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="searchPushButton">
     <property name="text">
      <string>Search</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QTableWidget" name="bltTableWidget"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
